lib/elixir/test/elixir/version_test.exs

```diff
@@ -111,6 +111,9 @@ defmodule VersionTest do
     assert V.match?("1.2.3-alpha", "1.2.3-alpha")
 
     assert V.match?("0.9.3", "== 0.9.3+dev")
+
+    {:ok, req} = Version.parse("2.3.0")
```

Comment:
It'd be better to rename `req` to `vsn`.
Requirement is a second argument for `match?` function, the first – version.